packages/web/src/components/nav/desktop/AccountSwitcher/AccountListContent.tsx

```diff
@@ -0,0 +1,103 @@
+import { useCallback } from 'react'
+
+import { ID, ManagedUserMetadata, UserMetadata } from '@audius/common/models'
+import { Box, Flex, IconUserArrowRotate, Text } from '@audius/harmony'
+import styled from '@emotion/styled'
+
+import { AccountSwitcherRow } from './AccountSwitcherRow'
+
+const messages = {
+  switchAccount: 'Switch Account',
+  maangedAccounts: 'Managed Accounts'
+}
+
+export type AccountListContentProps = {
+  accounts: ManagedUserMetadata[]
+  managerAccount: UserMetadata
+  currentUserId: ID
+  onAccountSelected: (user: UserMetadata) => void
+}
+
+const StyledList = styled.ul`
```

Comment:
just curious, why not use `css` prop on a `ul`? im probably out of the loop but didn't know we were trying to use `styled` in the codebase these days

Reply:
I asked this question in Slack and got what I thought was a suggestion to use emotion/styled as we're starting to use that pattern in Harmony. It was more of a personal choice of not wanting to bury the styling in a component prop. But I could be convinced to do it with the prop if there's an advantage to it.